### PR TITLE
register service for tests

### PR DESCRIPTION
**Pull request by rkondratowicz**
in order to run the tests, there needs to be a service configured in the application.

Tests using it to follow.

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -6,11 +6,20 @@ locals {
   preview_vault_name     = "${var.raw_product}-aat"
   default_vault_name     = "${var.raw_product}-${var.env}"
   vault_name             = "${(var.env == "preview" || var.env == "spreview") ? local.preview_vault_name : local.default_vault_name}"
+
+  preview_vault_rg       = "${var.raw_product}-aat"
+  default_vault_rg       = "${var.raw_product}-${var.env}"
+  vault_rg               = "${(var.env == "preview" || var.env == "spreview") ? local.preview_vault_rg : local.default_vault_rg}"
```
Comment (rkondratowicz) [inline infrastructure/main.tf]: previously pull request would create(or look for) vault in its own resource group. This fixes it.

File: infrastructure/main.tf [hunk 2]
```diff
@@ -6,11 +6,20 @@ locals {
...
 
 # Make sure the resource group exists
 resource "azurerm_resource_group" "rg" {
-  name     = "${var.product}-${var.component}-${var.env}"
+  name     = "${var.product}-${var.env}"
+  location = "${var.location}"
+}
+
+resource "azurerm_resource_group" "vault_rg" {
+  name     = "${local.preview_vault_rg}"
```
Comment (lgonczar) [inline infrastructure/main.tf]: This should be `${local.vault_rg}`, I think.

File: infrastructure/main.tf [hunk 3]
```diff
@@ -61,7 +71,12 @@ module "key-vault" {
   env                 = "${var.env}"
   tenant_id           = "${var.tenant_id}"
   object_id           = "${var.jenkins_AAD_objectId}"
-  resource_group_name = "${azurerm_resource_group.rg.name}"
+  resource_group_name = "${azurerm_resource_group.vault_rg.name}"
```
Comment (lgonczar) [inline infrastructure/main.tf]: Should it not stay as it was?

File: infrastructure/main.tf (hunk 1 above)
Comment (doncem) [reply infrastructure/main.tf]: isn't that exactly the same as `vault_name` block definition few lines above?

Comment (rkondratowicz) [reply infrastructure/main.tf]: After the change in line 17 it happens to be the same.
But the name of the vault and the resource group that it's in are two separate things and can be named differently.

**Pull request merged**